### PR TITLE
Carlos1530gamer/ar scene view controller/add first state

### PR DESCRIPTION
**Pull request by Carlos1530gamer**

### CODE REVIEW

File: Santillana-Demo-AR-Experiencia-Residuos/ARSceneViewConstans.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+//  Created by Carlos Daniel Hernandez Chauteco on 6/13/19.
+//  Copyright © 2019 Alejandro Mendoza. All rights reserved.
+//
+
+import Foundation
+
+class ARSceneViewConstans {
+    static let share = ARSceneViewConstans()
+    
+    let labelPlaneDetectionFirstMessage = "Escanea tu cuarto para iniciar"
```
Comment (zippi-MD) [inline Santillana-Demo-AR-Experiencia-Residuos/ARSceneViewConstans.swift:14]: Can you change the constants names? I think something like 'startScanningMessage' would be more suitable

File: Santillana-Demo-AR-Experiencia-Residuos/ARSceneViewController.swift [hunk 2]
```diff
@@ -9,17 +9,88 @@
...
+    @IBAction func stopPlaneDetection(_ sender: UIButton) {
+        let config = ARWorldTrackingConfiguration()
+        
+        self.arSceneView.session.run(config)
+        
+        for child in debugPlanes{ //clear the debug planes
+            child.opacity = 0.0
+        }
+        
+        UIView.transition(with: detectingPlanesLabel, duration: TimeInterval(3), options: .transitionCrossDissolve, animations: {[unowned self] in
```
Comment (zippi-MD) [inline Santillana-Demo-AR-Experiencia-Residuos/ARSceneViewController.swift:77]: We still haven't discussed animations...

File: Santillana-Demo-AR-Experiencia-Residuos/ARSceneViewController.swift [hunk 3]
```diff
@@ -35,31 +106,30 @@ class ARSceneViewController: UIViewController {
...
         
         arSceneView.scene = scene
-        
-        setupPicker()
     }
     
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
+    //MARk: Observer methods for state of game
+    private func detectingPlanesSetupUI(){
```
Comment (zippi-MD) [inline Santillana-Demo-AR-Experiencia-Residuos/ARSceneViewController.swift:112]: I don't believe we need private functions here, this View Controller is not used by anyone

**Review by eduardoquintero117 — APPROVED** [review]

File: Santillana-Demo-AR-Experiencia-Residuos/ARSceneViewController.swift [hunk 4]
```diff
@@ -35,31 +106,30 @@ class ARSceneViewController: UIViewController {
...
     
-
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
+    private func retriveInfoSetupUI(){
```
Comment (zippi-MD) [inline Santillana-Demo-AR-Experiencia-Residuos/ARSceneViewController.swift:130]: Same here, why this function is private and the others don't?

**Review by zippi-MD — CHANGES_REQUESTED** [review]

**Pull request closed without merging**